tests/filecheck/dialects/vector/vector_fma_invalid.mlir

```diff
@@ -0,0 +1,49 @@
+// RUN: xdsl-opt %s --verify-diagnostics --split-input-file | filecheck %s
```

Comment:
Is this file just the merging of the deleted files?

Reply:
Yep and adding one new one for the fact that it's a float